newsroom/agenda/views.py

```diff
@@ -453,13 +461,14 @@ def gen_agg_terms(field: str):
         ]:
             country_name = country_bucket["key"]
             for state_bucket in country_bucket["states"]["buckets"]:
-                regions.append(
-                    {
-                        "name": state_bucket["key"],
-                        "country": country_name,
-                        "type": "state",
-                    }
-                )
+                if state_bucket["key"]:
```

Comment:
can you try to add `exlude: [""],` to the `gen_agg_terms` method so it won't be returned from elastic,
[should be possible to filter it out there](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-terms-aggregation.html#_filtering_values_with_exact_values_2) and make it work for all fields